### ISSUE TITLE
is there a way to save results to disk when reboot?

### ISSUE THREAD

**Issue by kolinfluence** [bits-and-blooms/bloom#92]
is there a way to save results to disk when reboot?

**Comment by kaashmonee** [comment]
yeah it seems like you can call WriteTo and define a file to write the binary contents to

https://github.com/bits-and-blooms/bloom/blob/1b8b697ca6b5ad9d5599ab8926fb997ccca0c23c/bloom.go#L355-L366
